test_runner/regress/test_pageserver_generations.py

```diff
@@ -492,7 +492,7 @@ def evict_all_layers(env: NeonEnv, tenant_id: TenantId, timeline_id: TimelineId)
     )
     client = env.pageserver.http_client()
     for layer in initial_local_layers:
-        if "ephemeral" in layer.name:
+        if "ephemeral" in layer.name or "temp" in layer.name:
```

Comment:
This is an intentionally inexact condition, so that if we change something about how we do temp suffixes in future, we won't have to update the test.